app/controllers/sessions_controller.rb

```diff
@@ -1,9 +1,22 @@
+require 'rqrcode'
+require 'erb'
+
 class SessionsController < Devise::SessionsController
+  include ControllerConcern
+  include ERB::Util
```

Comment:
Can we remove this include and the two requires above?

Reply:
removed